lib/dune

```diff
@@ -0,0 +1,5 @@
+(library
+  (name multipart_form_data)
+  (public_name multipart-form-data)
+  (libraries lwt stringext)
```

Comment:
I'd rather keep the one lib per line form if you don't mind, this makes the diffs easier to read when it's updated!

Same applies to `pps` and in `tests/dune` of course.